style.css

```
@@ -198,41 +203,161 @@
   font-size: 12px;
 }
 
-@media (max-width: 700px) {
-  .hero-text__heading {
-    font-size: 20px;
-  }
+/* ------------courses page----- */
 
-  .nav-links__item {
-    display: block;
-  }
+.content-section {
+  padding: 4% 4%;
```

```suggestion
  padding: 4%;
```

no difference, just making sure you know that option